public/js/controllers/invitePlayers.js

```diff
@@ -14,6 +14,10 @@ angular.module('mean.system')
     $scope.searchHelper = '';
     var token = localStorage.getItem('#cfhetusertoken');
 
+    $scope.closeReusedModal = function () {
+      $('#reuse-modal').modal('close');
+      game.isFilledUp = null;
```

Comment:
Why null and not false

Reply:
This aspect is not my code, I had to move it to invitePlayers.js for the test to pass. Ikechukwu and Ben's Work.